elastic/kibana
# [Security Solution] Fix topN grouping when field type is boolean

## Summary

https://github.com/elastic/kibana/issues/131216

Before:
The legend displays 0, 1 when field type is boolean.

https://user-images.githubusercontent.com/61860752/165902576-4a6c3bbf-bc66-40cc-a496-d2f879659ef3.mp4


After:
The legend displays true, false when field type is boolean.

https://user-images.githubusercontent.com/6295984/167862198-04659866-4693-4415-b325-ef8fd73a7b65.mp4



Steps to verify:


1. Create a rule and generate some alerts.
2. Find the alerts table, click on `fields`, click `Create field` and create a runtime field for testing.
3. Name `process_interactive`, type `Boolean`, `Set value`: ```emit(true)```
4. Close the modal, hover onto any `process.interactive` cell, expand the hover actions and click `Show top values`
5. The legend should display true as group name instead of number.
6. Check the topN in timeline and alerts table as well.
7. Check topN for other field and make sure they display correctly as well.


### Checklist

Delete any items that are not applicable to this PR.
